src/Twig/CitizenProjectRuntime.php

```diff
@@ -58,4 +58,14 @@ public function canSeeCitizenProject(CitizenProject $citizenProject): bool
     {
         return $this->authorizationChecker->isGranted(CitizenProjectPermissions::SHOW, $citizenProject);
     }
+
```

Comment:
Not related to your PR but is it valuable to have a nullable argument in the constructor?

Reply:
I don't think so.

Reply:
fixed